keel-bakery-history-sql/src/test/kotlin/com/netflix/spinnaker/keel/bakehistory/sql/spring/SpringStartupTests.kt

```diff
@@ -0,0 +1,42 @@
+package com.netflix.spinnaker.keel.bakehistory.sql.spring
+
+import com.netflix.spinnaker.keel.KeelApplication
+import com.netflix.spinnaker.keel.bakehistory.sql.SqlBakeHistory
+import com.netflix.spinnaker.keel.bakery.artifact.BakeHistory
+import org.junit.jupiter.api.Test
+import org.junit.jupiter.api.extension.ExtendWith
+import org.springframework.beans.factory.annotation.Autowired
+import org.springframework.boot.test.context.SpringBootTest
+import org.springframework.boot.test.context.SpringBootTest.WebEnvironment.MOCK
+import org.springframework.test.context.junit.jupiter.SpringExtension
+import strikt.api.expectThat
+import strikt.assertions.isA
+
+@ExtendWith(SpringExtension::class)
+@SpringBootTest(
+  classes = [KeelApplication::class],
+  webEnvironment = MOCK,
+  properties = [
+    "keel.plugins.bakery.enabled=true",
+    "sql.enabled=true",
+    "orca.enabled=true",
+    "orca.base-url=http://localhost:8087",
+    "echo.base-url=http://localhost:8083",
+    "clouddriver.enabled=true",
+    "clouddriver.base-url=http://localhost:8085",
+    "igor.base-url=http://localhost:8084",
```

Comment:
I'd like to figure out if I can avoid having to set all this up

Reply:
We could just use an `application.properties` or `keel.yml` for tests, right?
You can use `@TestPropertySource` to specify the location: https://docs.spring.io/spring/docs/current/javadoc-api/org/springframework/test/context/TestPropertySource.html

Reply:
Damn, I had totally missed that this was why I wasn't having the same problems in `keel-sql` or `keel-web`. Thanks. Good spot.